File: SafeDalat_API/SafeDalat_API/Model/DTO/Incident/IncidentSearchDTO.cs
namespace SafeDalat_API.Model.DTO.Incident
{
    public class IncidentSearchDTO
    {
        public string? Keyword { get; set; }
        public string? Ward { get; set; }
        public int? CategoryId { get; set; }
        public int? AlertLevel { get; set; }

        // Tìm theo vị trí
        public double? Latitude { get; set; }
        public double? Longitude { get; set; }
        public double? RadiusKm { get; set; }

        public int PageIndex { get; set; } = 1; // Trang hiện tại (Mặc định 1)
        public int PageSize { get; set; } = 10;
    }
}

File: DalatS/app/src/main/java/com/example/dalats/model/IncidentSearchDTO.java
package com.example.dalats.model;

public class IncidentSearchDTO {
    private String keyword;
    private String ward;
    private Integer categoryId;
    private int pageIndex;
    private int pageSize;

    public IncidentSearchDTO(String keyword, String ward, Integer categoryId, int pageIndex, int pageSize) {
        this.keyword = keyword;
        this.ward = ward;
        this.categoryId = categoryId;
        this.pageIndex = pageIndex;
        this.pageSize = pageSize;
    }
}